include/testsuite.h

```diff
@@ -0,0 +1,56 @@
+void test_main_aes(void);
```

Comment:
Please add include guards.

```suggestion
#ifndef APP_LIBNETTLETEST_TESTSUITE_H
#define APP_LIBNETTLETEST_TESTSUITE_H
void test_main_aes(void);
```